test/start-operator.sh

```diff
@@ -75,13 +75,18 @@ function deploy_using_olm() {
 }
 
 function deploy_using_yaml() {
-  crd=${REPO_DIRECTORY}/deploy/crd/pmem-csi.intel.com_deployments_webhook.yaml
+  crd=${REPO_DIRECTORY}/deploy/crd/pmem-csi.intel.com_pmemcsideployments.yaml
+  echo "Deploying '${crd}'..."
+  cat ${crd} | ${SSH} kubectl apply -f -
+
+  # v1beta1 PmemCSIDeploymentCRD with conversion webhook configuration
+  crd=${REPO_DIRECTORY}/deploy/crd/pmem-csi.intel.com_pmemcsideployments_webhook.yaml
```

Comment:
I'm getting an error about `pmem-csi.intel.com_pmemcsideployments_webhook.yaml` not existing when running E2E tests for the operator.

Reply:
Can you also remove the code which install cert-manager? We don't need it at the moment.

Reply:
Put the remove of the cert-manager in a separate commit, in case that we want to bring it back without the operator conversion webhook.

Reply:
> I'm getting an error about `pmem-csi.intel.com_pmemcsideployments_webhook.yaml` not existing when running E2E tests for the operator.

My bad, This line shouldn't be there. I will fix it.